test/integration/history_test.go

```diff
@@ -67,8 +68,14 @@ func TestHistory(t *testing.T) {
 
 		changes, err := adminCli.ListChangeSummaries(ctx, "default", d1.Key(), 0, 0, true)
 		assert.NoError(t, err)
-		assert.Len(t, changes, 3)
 
+		if helper.SnapshotWithPurgingChanges {
+			// There is no change when SnapshotInterval is not set.
+			assert.Len(t, changes, 0)
+			return
+		}
```

Comment:
Is this code ever actually executed?

Reply:
I added this in case the developer sets SnapshotWithPurgingChanges in helper.go to true and run test.
But on second thought, I think it would be better to add a separate test case for test automation..!